clang/test/dpct/module_kernel.cu

```diff
@@ -23,36 +22,51 @@ __constant__ unsigned int const_data[3] = {1, 2, 3};
 //CHECK-NEXT: int a = item_ct1.get_local_id(2);
 //CHECK-NEXT: }
 
-//CHECK: extern "C" {
-//CHECK-NEXT: void foo_wrapper(sycl::queue &queue, const sycl::nd_range<3> &nr, unsigned int localMemSize, void **kernelParams, void **extra){
-//CHECK-NEXT: float * k;
-//CHECK-NEXT: float * y;
-//CHECK-NEXT: if(kernelParams){
-//CHECK-NEXT:  k = (float *)kernelParams[0];
-//CHECK-NEXT:  y = (float *)kernelParams[1];
-//CHECK-NEXT: } else {
-//CHECK-NEXT:  k = (float *)(extra + sizeof(void*));
-//CHECK-NEXT:  y = (float *)(extra + sizeof(void*) + sizeof(float *));
-//CHECK-NEXT: }
-//CHECK-NEXT: queue.submit(
-//CHECK-NEXT:  [&](sycl::handler &cgh) {
-//CHECK-NEXT:    const_data.init(queue);
-//CHECK:    auto const_data_ptr_ct1 = const_data.get_ptr();
-//CHECK:    sycl::local_accessor<uint8_t, 1> dpct_local_acc_ct1(sycl::range<1>(localMemSize), cgh);
-//CHECK:    cgh.parallel_for(
-//CHECK-NEXT:      nr,
-//CHECK-NEXT:      [=](sycl::nd_item<3> item_ct1) {
-//CHECK-NEXT:        foo(k, y, item_ct1, dpct_local_acc_ct1.get_pointer(), const_data_ptr_ct1);
-//CHECK-NEXT:      });
-//CHECK-NEXT:  });
-//CHECK-NEXT: }
-//CHECK-NEXT: }
+
+// CHECK:      extern "C" {
+// CHECK-NEXT:   DPCT_EXPORT void foo_wrapper(sycl::queue &queue, const sycl::nd_range<3> &nr, unsigned int localMemSize, void **kernelParams, void **extra) {
+// CHECK-NEXT:     dpct::args_selector<2, 0, decltype(foo)> selector(kernelParams, extra);
```

Comment:
`<2, 0, decltype(foo)>` is difficult to understand. Please reference the design of kernel wrapper.

Reply:
May be mitigate by adding a comment:

// 2 default parameters, 0 non-default parameters
dpct::args_selector<2, 0, decltype(foo)> selector(kernelParams, extra);